Utility.py

```diff
@@ -0,0 +1,18 @@
+"These are simple utility functions"
+from numpy import log10, ndarray, array
+def dBm2power(dBm):
+    "convert from dBm to W"
+    if type(dBm) is not ndarray:
+        dBm = array(dBm)
+    return 10**(0.1*dBm-3)
+def power2dBm(power):
+    "convert from dBm to W"
+    if type(power) is not ndarray:
+        power = array(power)
+    return log10(power)*10+30
+def Vpp2power(Vpp,Z=50.):
+    "convert peak to peak voltage to power"
+    if type(Vpp) is not ndarray:
+        Vpp = array(Vpp)
+    Vrms = Vpp/2/sqrt(2)
+    return Vrms**2/Z
```

Comment:
this file should be inside the package

Reply:
done

Reply:
looks like you copied, rather than just moving?

Reply:
I moved using my file explorer and then added the new document in on gitbash

Reply:
hmm.. if you followed this with a `git commit -a`, I would have thought that would work.  But, anyways, in the future the best way to move files under git control is using `git mv` inside bash.

to fix this problem, just `git rm` the file in its original location (repo main directory), if there is a file there, or else `git commit -a`

Reply:
okay. Noted. I do not see Utility inside the proc_scripts but do see it in proc_scripts/proc_scripts. correct me if I am wrong